### ISSUE TITLE
Camera Parameters/Homography? 

### ISSUE THREAD

**Issue by harshap-ai** [SoccerNet/sn-tracking#16]
Hello authors/curators, 

Do we have corresponding camera parameters for these videos, by any chance? Or any homography information?

**Comment by SilvioGiancola** [comment]
Yes, we extended the tracking task to a "game state reconstruction" https://github.com/SoccerNet/sn-gamestate that contains the same tracking sequences with extra annotations for player annotations, field annotations, and camera calibration.
